packages/flutter/lib/src/material/app_bar_theme.dart

```diff
@@ -126,6 +134,8 @@ class AppBarTheme with Diagnosticable {
       shadowColor: Color.lerp(a?.shadowColor, b?.shadowColor, t),
       iconTheme: IconThemeData.lerp(a?.iconTheme, b?.iconTheme, t),
       actionsIconTheme: IconThemeData.lerp(a?.actionsIconTheme, b?.actionsIconTheme, t),
+      // TODO(sidrao2006): Recheck implementation, https://github.com/flutter/flutter/pull/64789.
+      iconButtonSplashRadius: t < 0.5 ? a.iconButtonSplashRadius : b.iconButtonSplashRadius,
```

Comment:
would it make sense to use lerpDouble here?

Reply:
Thanks a lot Ian